bot.go

```diff
@@ -157,13 +159,14 @@ func onMessageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 		arr1 := strings.Split(m.Content, " ")
 		if len(arr1) == 2 && arr1[0] == "!albumcreate" {
 			CreateAlbum(arr1[1])
+			s.ChannelMessageSend(m.ChannelID, arr1[1]+"というアルバムを作成したよ！")
```

Comment:
今回の変更とは直接関係ないけどここもerr見るようにしておいてもらえますか？

```suggestion
			err := CreateAlbum(arr1[1])
			if err != nil {
				return err
			}
			s.ChannelMessageSend(m.ChannelID, arr1[1]+"というアルバムを作成したよ！")
```